aws_xray_sdk/sdk_config.py

```diff
@@ -27,13 +27,17 @@ class SDKConfig(object):
             to occur.
     """
     XRAY_ENABLED_KEY = 'AWS_XRAY_SDK_ENABLED'
-    __SDK_ENABLED = str(os.getenv(XRAY_ENABLED_KEY, 'true')).lower() != 'false'
+    DISABLED_ENTITY_NAME = 'dummy'
+
+    __SDK_ENABLED = None
 
     @classmethod
     def sdk_enabled(cls):
         """
         Returns whether the SDK is enabled or not.
         """
+        if cls.__SDK_ENABLED is None:
+            cls.__SDK_ENABLED = str(os.getenv(cls.XRAY_ENABLED_KEY, 'true')).lower() != 'false'
```

Comment:
why not something like
```
from distutils.util import strtobool
#...
            cls.__SDK_ENABLED = strtobool(os.getenv(cls.XRAY_ENABLED_KEY, 'true'))
```

`strtobool` source code:

https://github.com/python/cpython/blob/daf62627518ad97ce66a48c49496aa0573cf0731/Lib/distutils/util.py#L306-L319

Reply:
Great suggestion! I'll add that to the next revision.

Reply:
okay, `strtobool` lie because it returns 1 and 0, not True or False, and this can lead to errors because `1 is True` is false.

Maybe
```
            cls.__SDK_ENABLED = bool(strtobool(os.getenv(cls.XRAY_ENABLED_KEY, 'true')))
```

Reply:
Done